src/components/Footer/Footer.jsx

```diff
@@ -1,5 +1,38 @@
 import * as S from './Footer.style';
 
 export default function Footer() {
-  return <S.Footer>footer</S.Footer>;
+  return (
+    <S.FooterWrapper>
+      <S.Footer>
+        <S.Logo>
+          <img src="/images/logo/logo-white-footer.svg" alt="APPS 로고" />
+        </S.Logo>
+
+        <S.DeveloperSection>
+          <S.SectionTitle>DEVELOPER</S.SectionTitle>
+          <S.TeamMemberContainer>
+            <S.TeamMember>김준희 (소프트웨어학부 22)</S.TeamMember>
+            <S.TeamMember>류미성 (소프트웨어학부 21)</S.TeamMember>
+            <S.TeamMember>이다빈 (소프트웨어학부 24)</S.TeamMember>
+            <S.TeamMember>하지민 (수학과, 소프트웨어학부 22)</S.TeamMember>
+          </S.TeamMemberContainer>
+        </S.DeveloperSection>
+
+        <S.DesignerSection>
+          <S.SectionTitle>DESIGNER</S.SectionTitle>
+          <S.TeamMemberContainer>
+            <S.TeamMember>김수진 (공예과 21)</S.TeamMember>
+            <S.TeamMember>문서영 (산업디자인과 22)</S.TeamMember>
+          </S.TeamMemberContainer>
+        </S.DesignerSection>
```

Comment:
```suggestion
        <S.TeamSection>
          <S.TeamCategory>
            <S.CategoryTitle>DEVELOPER</S.CategoryTitle>
            <S.TeamMemberContainer>
              <S.TeamMember>김준희 (소프트웨어학부 22)</S.TeamMember>
              <S.TeamMember>류미성 (소프트웨어학부 21)</S.TeamMember>
              <S.TeamMember>이다빈 (소프트웨어학부 24)</S.TeamMember>
              <S.TeamMember>하지민 (수학과, 소프트웨어학부 22)</S.TeamMember>
            </S.TeamMemberContainer>
          </S.TeamCategory>

          <S.TeamCategory>
            <S.CategoryTitle>DESIGNER</S.CategoryTitle>
            <S.TeamMemberContainer>
              <S.TeamMember>김수진 (공예과 21)</S.TeamMember>
              <S.TeamMember>문서영 (산업디자인과 22)</S.TeamMember>
            </S.TeamMemberContainer>
          </S.TeamCategory>
        </S.TeamSection>
```


- `TeamSection`으로 전체를 감싸주는 컴포넌트를 추가하고
- `DeveloperSection`와 `DesignerSection`을 `TeamCategory`으로 수정했어요

`TeamSection` 내부에 gap 속성 추가하고, 여기에 margin 속성을 주면 좋을 것 같아요 :)